### PR TITLE
Move types for node and http to 'dependencies' from 'devDependencies'

### LINKED ISSUE #156: Typescript definitions are broken in 0.7.1
The 0.7.1 release has broken the typescript definitions - running `tsc` on the example for listing all pods, I get numerous errors like
> node_modules/@kubernetes/client-node/dist/api.d.ts:13:26 - error TS7016: Could not find a declaration file for module 'request'. '/home/marina/kube-dep-test/node_modules/request/index.js' implicitly has an 'any' type.

The typings work in v0.6.0; v0.7.0 compiles successfully as well, but I get the runtime error described in #135.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes #156 (I think)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kubernetes/client-node",
-  "version": "0.7.2",
+  "version": "0.7.3",
```
[the remark this answers is not in the document]
Comment (brendandburns) [reply package.json]: Done.

**Comment by brendandburns** [comment]
@drubin comment addressed, I think this is ready to merge.

**Comment by flyboarder** [comment]
Im not sure this is needed see #156 thread - issue seems to be resolved

**Comment by drubin** [comment]
@brendanburns thanks. 

lgtm (not sure if I can run commands)

**Comment by brendandburns** [comment]
@drubin are you interested in becoming a contributor to this library? If so I can add you and you can run those commands (if you promise to review code :)

**Comment by drubin** [comment]
@brendandburns sure that would be great I would love to! 

I try to review the changes any way since we rely quite heavily on this library.

**Comment by brendandburns** [comment]
self lgtm on approve from @drubin (whom I'm adding to the org)

**Comment by drubin** [comment]
@brendandburns Thanks.

**Pull request merged**